### PR TITLE
[2.5]Remove k8s 1.20 restriction

### PR DESCRIPTION
**Pull request by StrongMonkey**

### CODE REVIEW

**Comment by StrongMonkey** [comment]
Remove k8s 1.20 restriction since it will be supported starting at 2.5.6.

**Comment by StrongMonkey** [comment]
https://github.com/rancher/rancher/issues/30134

**Review by luthermonson — DISMISSED** [review]

File: chart/Chart.yaml [hunk 1]
```diff
@@ -3,7 +3,6 @@ name: rancher
 description: Install Rancher Server to manage Kubernetes clusters across providers.
 version: %VERSION%
 appVersion: %APP_VERSION%
-kubeVersion: < 1.20.0-0
```
Comment (cbron) [inline chart/Chart.yaml:6]: Why not: `kubeVersion: < 1.21.0-0` ?

**Review by cbron — APPROVED** [review]

**Comment by cbron** [comment]
@StrongMonkey we should forwardport this to master also

**Pull request merged**